ui/components/User.js

```diff
@@ -160,6 +161,21 @@ const User = (props) => {
   const { color, iconButtonClassName, avatarClassName, classes } = props;
 
   const open = Boolean(anchorEl);
+
+
+  if ( userData?.status == 'anonymous' ) {
+    const url =
+         `${capabilitiesRegistry?.provider_url}/user/login?anonymousUserID=${userData?.user_id}&redirect=${window.location.pathname}`
+        
```

Comment:
// EXTENSION POINT: Anonymous Users
// Insert custom logic here to handle Single User mode, Anonymous User mode, Multi User mode behavior.